bundler/spec/commands/install_spec.rb

```diff
@@ -871,6 +871,36 @@ def run
     end
   end
 
+  describe "when bundle bin dir does not have write access", :permissions do
+    let(:bin_dir) { bundled_app("vendor/#{Bundler.ruby_scope}/bin") }
+
+    before do
+      FileUtils.mkdir_p(bin_dir)
```

Comment:
do you need to clean up this dir in an `after` block?

Reply:
I think not strictly needed because the global after block cleans up the `bundled_app` folder anyways. But it doesn't hurt to make the test more self-contained in my opinion, will add it!

Reply:
Actually, I see there are many other tests in this file that create some directory inside `bundled_app` and they don't clean up after themselves and rely on the global teardown for the cleanup. So I'll leave it as is for consistency. If we feel it's important enough, we can change everything on a separate PR.